corehq/apps/style/static/style/js/pagination.ng.js

```diff
@@ -0,0 +1,121 @@
+(function (angular, undefined) {
+    'use strict';
+
+    var pagination = angular.module('hq.pagination', [
+        'ui.bootstrap',
+        'ngResource',
+        'ngRoute',
+        'ngCookies',
+        'ng.django.rmi'
+    ]);
+
+    pagination.constant('paginationLimits', [
+        [10, "Limit 10"]
```

Comment:
The caller is expected to override this, yeah?  Otherwise this won't be translated.  I see that the mobile worker page does so, but I'm wondering if there's a way to force that.